carbon-design-system/carbon
# Update DAP to v2.0.6

Updating DAP to latest. v2.0.6

> Apparently there are settings in jest that override settimeout behavior, which basically causes any usage of settimeout to hang (supposed to be for testing ticks, but has collateral side effects). Shouldn't have been irregular on your end, but worth a shot. This version will skip not use the settimeout approach to getting out of the stack when in jest **environments.**

❤️ 

### To test
in root folder run `yarn test` and make sure DAP is being run without unexpected error.
